src/render_engine/themes.py

```diff
@@ -84,7 +84,7 @@ def register_theme(self, theme: Theme):
 
         if theme.template_globals:
             for key, value in theme.template_globals.items():
-                self.engine.globals.setdefault(key, set()).add(value if not isinstance(value, list) else tuple(value))
+                self.engine.globals.setdefault(key, set()).add(value if not isinstance(value, list) else (value))
```

Comment:
As discussed in the Discord, I'm a little concerned about this. I'd definitely like to seem some tests of what this looks like for plugin creators